src/app/pages/news/11.3/05-portal.md

```diff
@@ -1,16 +1,25 @@
 ## Axon Ivy Portal {#portal}
 
-... any news?
+Axon Ivy Portal 11.3 – your key to boosted productivity and seamless workflow! With an enhanced complex filter, lightning-fast search, customizable statistic charts, and real-time notifications, managing tasks and cases is effortless. Plus, now that the portal is now open source!
+
+- **Improved Complex Filters**: With the reinvented complex filter you can refine your task and caselist searches with ease. The new filter can combine an endless number of filter conditions. Various new filters for custom fields have been added, and even moving timeframes are now possible. So say goodbye to endless scrolling and hello to targeted results within the task and case widget of the portal dashboard.
+
+- **New Statistic Charts**: In this release we've overhauled and enhanced the statistics capabilities of the portal. We've introduced numerous new standard charts, and now, custom charts can also be tailored for your business users and seamlessly integrated into both standard and private dashboards. Experience unparalleled performance with process data being provided through the Elasticsearch engine.
+
+- **Notifications**: Effortlessly receive real-time updates through the new Notifications in the Axon Ivy Portal. Ensuring you're always in the know about crucial tasks and project milestones. You can tailor the notifications enabling a personalized experience that keeps you ahead of deadlines and boosts productivity with ease.
```

Comment:
the last sentence may be simplified: 
`You can personalize your notification subscriptions to boost your productivity.`